fuel-vm/src/util.rs

```diff
@@ -430,20 +447,40 @@ pub mod test_helpers {
                 .build()
         }
 
+        #[cfg(any(fuzzing, feature = "test-helpers"))]
+        pub fn setup_contract_bytes(
+            &mut self,
+            contract: Vec<u8>,
+            initial_balance: Option<(AssetId, Word)>,
+            initial_state: Option<Vec<StorageSlot>>,
+        ) -> CreatedContract {
+            self.setup_contract_inner(contract, initial_balance, initial_state)
+        }
+
         pub fn setup_contract(
             &mut self,
             contract: Vec<Instruction>,
             initial_balance: Option<(AssetId, Word)>,
             initial_state: Option<Vec<StorageSlot>>,
+        ) -> CreatedContract {
+            let contract = contract
+                .into_iter()
+                .flat_map(Instruction::to_bytes)
+                .collect();
```

Comment:
```suggestion
                .into_iter()
                .collect();
```

Reply:
Updated in https://github.com/FuelLabs/fuel-vm/pull/733/commits/86f73591b00cdc297ef4905c4ebd4b6ee0dc7d94